### ISSUE TITLE
Base64 decoding error in Safari

### ISSUE THREAD

**Issue by rhalukongor** [milankinen/livereactload#67]
When I try to use live react reload in Safari, I get the error:

data:application/json;base64,*BASE64 DATA HERE*;
Failed to load resource: The operation couldn’t be completed. (kCFErrorDomainCFNetwork error -10.)

[This stack overflow answer](http://stackoverflow.com/questions/14678971/safari-error-kcferrordomaincfnetwork-10) suggests that it is incorrect encoding that causes the error.

Firefox have no problems though.

**Comment by milankinen** [comment]
Could you describe the context where you are using that `data:application/json;base64,BASE64 DATA HERE;`?

**Comment by rhalukongor** [comment]
Sure: I've put up a very basic project to try out live-react-load. When I sh the following:
```
watchify \
            -v \
            -d \
            --extension=.jsx \
            -t babelify \
            -g [ envify --URL_PREFIX $UP_DEV ] \
            -o $OUT_FILE $IN_FILE
```
i have no problems; working source maps in Safari, no errors... However after adding live-react-load to the table:
```
watchify \
            -v \
            -d \
            --extension=.jsx \
            -t babelify \
            -p [ livereactload ] \
            -g [ envify --URL_PREFIX $UP_DEV ] \
            -o $OUT_FILE $IN_FILE
```
I get the error I've mentioned. Firefox doesn't throw any errors but some sources fail displaying an error. These are all happening at the load time by the way. Anything else you need?

**Comment by rhalukongor** [comment]
Seems the plugin adds source maps for itself even without the -d flag. Those maps also causes the same error (without watchify's -d flag) so it seems there is something wrong with the generated source maps of live-react-reload.

**Comment by milankinen** [comment]
Phew... This was a tough one... Had added accidentally one extra `;` to the end of the `sourceMapURL` which caused the error in Safari.

It also seems that Babel creates inline source maps by default and Browserify strips them away if debug flag is not set. However, because LiveReactload gets the live-reloadable sources just before that operation, the existing source maps will be included into the bundle.

So if you want to create bundle without source map, they should be explicitly turned of from Babel. However, they should not affect the Safari anymore after this fix: 34084737e105c641445b9a65f901e16355bd7de2

**Comment by milankinen** [comment]
Should be fixed in `2.0.2`

**Comment by rhalukongor** [comment]
I hereby confirm that it is fixed sir. Thanks for the almost immediate fix. Seems 2.0.2 introduced a new bug though; I'll open another issue for that.
